Sources/AuthFoundation/JWT/Enums/JWTClaim.swift

```diff
@@ -229,62 +232,24 @@ public enum Claim: Codable {
 
     /// Token introspection response
     case tokenIntrospection
-
-    /// Custom claim with the given name
-    case custom(_ name: String)
-}
-
-/// Used by classes that contains OAuth2 claims.
-///
-/// This provides common conveniences for interacting with user or token information within those claims. For example, iterating through ``allClaims-4c54a`` or using keyed subscripting to access specific claims.
-public protocol HasClaims {
-    /// Returns the collection of claims this object contains.
-    ///
-    /// > Note: This will only return the list of official claims defined in the ``Claim`` enum. For custom claims, please see the ``customClaims`` property.
-    var claims: [Claim] { get }
     
-    /// Returns the collection of custom claims this object contains.
-    ///
-    /// Unlike the ``claims`` property, this returns values as strings.
-    var customClaims: [String] { get }
+    /// Indicates whether the transaction is on a nonce-supported platform. If you sent a nonce in the authorization request but do not see the nonce claim in the ID token, check this claim to determine how to proceed. Used predominantly by Sign In With Apple.
```

Comment:
`platform` or `protocol`?

Reply:
This is referring to the Authorization Server platform (this is mostly copy/pasted from the OIDC spec itself).